<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color:rgb(252, 234, 255);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cosmetics_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IMG-20240730-WA0109.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="nails_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IMG-20240730-WA0112.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="hair_label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IMG-20240730-WA0110.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="cosmeticsBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cosmetics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nailsBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>490</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Nails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="meetUsBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Meet The Team</string>
    </property>
   </widget>
   <widget class="QPushButton" name="productsBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Our products</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pricesBtn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Prices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apptBtn">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Book an appointment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hairBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Hair</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
